alf/examples/oac_halfcheetah_conf.py

```diff
@@ -23,7 +23,7 @@
 from alf.utils.math_ops import clipped_exp
 from alf.utils.losses import element_wise_squared_loss
 
-from alf.examples import sac_conf
+import sac_conf
```

Comment:
Please revert all import changes to "from alf.examples import ...". It seems that your change is not based on the most update-to-date alf. Make sure to pull first.

Reply:
Reverted all import changes to "from alf.examples import ..."